puppet/zulip/files/cron.d/archive-messages

```diff
@@ -0,0 +1,10 @@
+MAILTO=root
+# Removing old archived data should follow after archiving messages,
+# cause it checks related archived messages and original attachments rows before deleting .
```

Comment:
Better phrasing, if I understand correctly:

~~~
# The retention policy data deletion process should remove old archived
# data *after* archiving messages, to check related archived messages
# and the original attachments' rows before deleting.
~~~